src/Common/tests/System/Net/Http/Http2Frames.cs

```diff
@@ -348,23 +348,24 @@ public class PingFrame : Frame
         public byte[] Data;
 
         public PingFrame(byte[] data, FrameFlags flags, int streamId) :
-            base(Frame.FrameHeaderLength + 8, FrameType.Ping, flags, streamId)
+            base(8, FrameType.Ping, flags, streamId)
```

Comment:
I'll leave this from this PR since we do that elsewhere and I'd prefer to avoid another merge conflict. We can address it separately next time we touch code in this file